oap-server/server-starter/src/main/resources/ui-initialized-templates/redis/redis-instance.json

```diff
@@ -151,26 +161,22 @@
           "h":6,
           "i":"5",
           "type":"Widget",
-          "metricTypes":[
-            "readMetricsValue"
-          ],
-          "metrics":[
-            "meter_redis_instance_memory_max_bytes"
-          ],
           "graph":{
             "type":"Card",
             "fontSize":14,
             "textAlign":"center",
             "showUnit":true
           },
-          "metricConfig":[
-            {
-              "calculation":"byteToMB"
-            }
-          ],
           "widget":{
             "title":"Memory Max Bytes (MB)"
-          }
+          },
+          "metricMode":"Expression",
+          "expressions":[
+            "meter_redis_instance_memory_max_bytes/1000/1000"
```

Comment:
```suggestion
            "meter_redis_instance_memory_max_bytes/1024/1024"
```

Reply:
Because this metric is special, it defaults to 100000000000, so I set it up this way.